src/layout/layout.js

```diff
@@ -401,13 +401,15 @@ function MaterialLayoutTab(tab, tabs, panels, layout) {
     }
 
     tab.addEventListener('click', function(e) {
-      e.preventDefault();
-      var href = tab.href.split('#')[1];
-      var panel = layout.content_.querySelector('#' + href);
-      layout.resetTabState_(tabs);
-      layout.resetPanelState_(panels);
-      tab.classList.add(layout.CssClasses_.IS_ACTIVE);
-      panel.classList.add(layout.CssClasses_.IS_ACTIVE);
+      if (tab.getAttribute('href').charAt(0) === '#') {
```

Comment:
@addyosmani @sgomes WDYT? I’m fine with landing as-is for now and covering that corner-case in documentation.